### ISSUE TITLE
Video not having length

### ISSUE THREAD

**Issue by vshalgarg** [LironHazan/ng-webrtc-demo#2]
Hello,
Is there some way to find the max length of the video. So that while playing, player can show the seekbar appropriately.

**Comment by LironHazan** [comment]
I'll check it out and push a fix asap thanks!

**Comment by vshalgarg** [comment]
Hi @LironHazan ,
Any update on this.

**Comment by LironHazan** [comment]
@vshalgarg not yet, I need to dig a bit into the api ( I first thought you're asking about css but I assume you want the video length as output)

**Comment by vshalgarg** [comment]
Yes @LironHazan, I want the video length as output so that player can recognize it and show the seekbar appropriately.

**Comment by vshalgarg** [comment]
any update @LironHazan
